### PR TITLE
test: Add compatibility check for AMB

### PR DESCRIPTION
**Pull request by kulla**

### CODE REVIEW

File: server/src/metadata.rs [hunk 1]
```diff
@@ -352,7 +352,7 @@ pub mod entities_metadata_query {
             "article" => "Article",
             "course" => "Course",
             "text-exercise-group" | "text-exercise" => "Quiz",
-            "video" => "Video",
+            "video" => "VideoObject",
```
Comment (AndreasHuber) [inline server/src/metadata.rs:355]: The test is still expecting "Video".

File: scripts/check_metadata_api.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+TMP_DIR=$(mktemp -tdu serlo-database-layer-check-metadata.XXXXXXXXXX)
+ZIP_FILE="$TMP_DIR/amb.zip"
+SCHEMAS_PATH="amb-main/draft/schemas/"
+SCHEMAS_DIR="$TMP_DIR/$SCHEMAS_PATH"
+SCHEMA="${SCHEMAS_DIR}schema.json"
+
+function init() {
+  set -e
+  trap 'tear_down' EXIT
+
```
Comment (AndreasHuber) [inline scripts/check_metadata_api.sh:16]: > Usage Error: Couldn't find the node_modules state file - running an install might help (findPackageLocation)

Do we maybe have to add a `yarn install`?

**Review by AndreasHuber — CHANGES_REQUESTED** [review]

File: scripts/check_metadata_api.sh (hunk 2 above)
Comment (kulla) [reply scripts/check_metadata_api.sh:16]: You have to run `yarn` in order to install all dependencies

**Comment by kulla** [comment]
test fixed. ready to be merged (ping @AndreasHuber )

File: scripts/check_metadata_api.sh (hunk 2 above)
Comment (kulla) [reply scripts/check_metadata_api.sh:16]: done!

**Review by AndreasHuber — APPROVED** [review]

**Pull request merged**